### PR TITLE
Hmt 7 judge showcase

### PR DESCRIPTION
**Pull request by burtonjong**
Just need some feedback on this component. Since the name/company are beside the image it was hard to find a fitting ratio for the image/font-size (as they would overlap when the screen becomes smaller). Let me know if I should change anything or if it looks off because on mobile it does look a little small in my opinion, but if it were made any larger, if the name of the person is long enough then it would overlap with the adjacent judge. 
Mobile View:
image
Desktop view below

Also, I made this component so that it fits the amount of judges (in this picture 9 judges) but should I make it so that it is 100% of the viewport width? As you can see the judging criteria is also in the viewport.
image

Also because I am not the best with Next/Contentful images I am not sure what the type of image is (string, static image type, etc) so this is pretty default until we connect it to Contentful. 
image

### CODE REVIEW

**Comment by burtonjong** [comment]
I also just realized I set enableLandingPage to false. I can change this before merging 
image

File: public/images/imgplaceholder.png
Comment (ideen1) [inline public/images/imgplaceholder.png:1]: 11/10 Approved

**Review by ideen1 — CHANGES_REQUESTED** [review]
Looks good to me just, need to set the enable Landing Page part to true :)

File: public/images/imgplaceholder.png
Comment (burtonjong) [reply public/images/imgplaceholder.png:1]: Haha thanks

**Comment by burtonjong** [comment]
Just made the feature flag change

**Review by ideen1 — APPROVED** [review]

**Comment by burtonjong** [comment]
There were merge conflicts due to page.tsx being changed from the "AboutEventTile" being merged into main. I resolved the conflicts in the github editor but for some reason it gave me an ESLint error..? So I tried rebasing from main to manually resolve the errors and yet there is still an ESLint error. 
image
Is there a reason why I should switch the positions of these imports just because of the prettier extension?? Makes no sense to me but I did and I guess it fixed it... Hopefully I did not break something

**Comment by ideen1** [comment]
> There were merge conflicts due to page.tsx being changed from the "AboutEventTile" being merged into main. I resolved the conflicts in the github editor but for some reason it gave me an ESLint error..? So I tried rebasing from main to manually resolve the errors and yet there is still an ESLint error. image Is there a reason why I should switch the positions of these imports just because of the prettier extension?? Makes no sense to me but I did and I guess it fixed it... Hopefully I did not break something

Hey Burton, the reason is that it’s trying to sort them according to a standard order across all files. In this case the component imports are sorted alphabetically. I agree that is has no functional difference but it helps keep the code base clean. These should be auto formatted on save. You can also run format in VSCode and it should do all these swaps for you.

**Review by ideen1 — APPROVED** [review]
Looks great!

**Pull request merged**